examples/directml/phi_1/phi_1_config.json

```diff
@@ -0,0 +1,78 @@
+{
```

Comment:
The nomenclature in the directml examples folder is to prefix the `config` instead of having it as a suffix.